index.js

```diff
@@ -87,6 +91,12 @@ function logError(context, error){
 }
 
 (async () => {
+    let stylesheet = await fs.readFile(__dirname + "/style.css", "binary");
+    if (argv.stylesheet) {
```

Comment:
I think you forgot to push your last changes to handle the arg

Reply:
hmm but isnt the arg handled in this very if?

Reply:
if you could provide some guidance as to what i should change, that would be awesome.